test/org/sagebionetworks/bridge/validators/AppConfigValidatorTest.java

```diff
@@ -55,16 +84,94 @@ public void surveyReferencesHaveCreatedOnTimestamps() {
         SurveyReference surveyRef = new SurveyReference("identifier","guid",null);
         appConfig.getSurveyReferences().add(surveyRef);
         
-        assertValidatorMessage(NEW_VALIDATOR, appConfig, "surveyReferences[0].createdOn", "is required");
+        assertValidatorMessage(newValidator, appConfig, "surveyReferences[0].createdOn", "is required");
     }
     
     @Test
     public void schemaReferencesHaveRevision() { 
         AppConfig appConfig = AppConfig.create();
+        appConfig.setStudyId(TestConstants.TEST_STUDY_IDENTIFIER);
         SchemaReference schemaRef = new SchemaReference("guid",null);
         appConfig.getSchemaReferences().add(schemaRef);
         
-        assertValidatorMessage(NEW_VALIDATOR, appConfig, "schemaReferences[0].revision", "is required");
+        assertValidatorMessage(newValidator, appConfig, "schemaReferences[0].revision", "is required");
+    }
+    
+    @Test
+    public void schemaDoesNotExistOnCreate() {
+        when(schemaService.getUploadSchema(any(), eq("guid"))).thenThrow(new EntityNotFoundException(AppConfig.class));
```

Comment:
Is it worth using eq("whatever") instead of any() to make sure we're passing through the args correctly? (Here and in all the other tests.) This is probably more important for verifying survey guid and createdOn are passed correctly.

Reply:
Definitely. Changed all of these to concrete arguments and cleaned up the redundancy so the tests are shorter and clearer.